tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "moduleResolution": "node",
     "lib": [
       "es2016",
-      "es2016.array.include"
+      "es2016.array.include",
+      "dom"
```

Comment:
Could you please explain this change?

Reply:
It's need for fix build errors https://github.com/reportportal/agent-js-playwright/runs/3518696676